lib/utils/resolve-error-location.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+const anonymizeStacktracePaths = require('./anonymize-stacktrace-paths');
+const tokenizeException = require('./tokenize-exception');
+
+const resolveErrorLocation = (error) => {
+  const exceptionTokens = tokenizeException(error);
+
+  if (exceptionTokens.isUserError && error.code) return null;
+
+  if (!exceptionTokens.stack) return null;
+
+  const splittedStack = exceptionTokens.stack.split('\n');
+  if (splittedStack.length === 1 && error.code) return null;
+
+  const stacktraceLineRegex = /\s*at.*\((.*)\).?/;
+  const stacktracePaths = [];
+  for (const line of splittedStack) {
+    const match = stacktraceLineRegex.exec(line) || [];
+    const matchedPath = match[1];
+    if (matchedPath) {
+      stacktracePaths.push(matchedPath);
+      // Limited to maximum 5 lines in location
```

Comment:
I've made arbitrary decision to trim the stack to at most 5 paths to not send potentially long traces. It should be more than enough to inspect the issue at hand.

Reply:
That's fine. In gist that was part of spec it was proposed to trim to 3 lines, but I already was afraid that can be too little. 

Let's see what we will get with 5 lines